docs/detections/detection-engine-intro.asciidoc

```diff
@@ -51,20 +51,26 @@ To make sure you can access Detections and manage rules, see
 
 [float]
 [[cold-tier-detections]]
-== Compatibility with cold and frozen tier nodes
+== Manage data in cold and frozen tiers
 
-Cold and frozen {ref}/data-tiers.html[data tiers] hold time series data that is only accessed occasionally. In {stack} version >=7.11.0, {elastic-sec} supports cold but not frozen tier data for the following {es} indices:
+Cold {ref}/data-tiers.html[data tiers] store time series data that's accessed infrequently and rarely updated, while frozen data tiers hold time series data that's accessed even less frequently and never updated. If you're automating searches across different data tiers using rules, consider the following best practices and limitations.
 
-* Index patterns specified in `securitySolution:defaultIndex`
-* Index patterns specified in the definitions of detection rules, except for indicator match rules 
-* Index patterns specified in the data sources selector on various {security-app} pages
+[float]
+[[best-practices-data-tiers]]
+=== Best practices 
+
+* **Retention in hot tier**: We recommend keeping data in the hot data tier ({ilm-cap} hot phase) for at least 24 hours. {ilm-cap} policies that move ingested data from the hot phase to another phase (for example, cold or frozen) in less than 24 hours may cause performance issues and/or rule execution errors.
+* **Replicas for mission-critical data**: Your data should have replicas if it must be highly available. Since frozen tiers don't support replicas, shard unavailability can cause partial rule run failures. Shard unavailability may be also encountered during or after {stack} upgrades. If this happens, you can <<manually-run-rules,manually rerun>> rules over the affected time period once the shards are available.
 
-{elastic-sec} does *NOT* support either cold or frozen tier data for the following {es} indices:
+[float]
+[[limitations-data-tiers]]
+=== Limitations
 
-* Index patterns controlled by {elastic-sec}, including alerts and list indices
-* Index patterns specified in the definition of indicator match rules 
+Data tiers are a powerful and useful tool. When using them, keep the following limitations in mind:
 
-Using either cold or frozen tier data for unsupported indices may result in detection rule timeouts and overall performance degradation.
+* {ilm-cap} policies for indices controlled by {elastic-sec}, including alerts and list indices, must not be modified.
```

Comment:
```suggestion
* You cannot modify {ilm-cap} policies for indices controlled by {elastic-sec}, including alerts and list indices.
```
This suggestion could be wrong, but as written before it sounded like you _could_ but shouldn't modify them. I'm not sure which is true, but if it's a "can but shouldn't" situation, I wonder why I shouldn't! Maybe we could explain it in brief?

Reply:
What if we go in the direction of encouraging users to work with the default ILM settings? For example:

```suggestion
* {ilm-cap} policies for {elastic-sec}-controlled indices, such as alert and list indices, function best with their default settings and should not be modified.
```

Reply:
We've gotten many SDHs where rules were broken due to modifications to ILM policies so I'd prefer stronger language, e.g. "modifying ILM policies for elastic controlled indices may cause rule failures"

Reply:
How's this, @marshallmain?

```suggestion
* To avoid rule failures, do not modify {ilm-cap} policies for {elastic-sec}-controlled indices, such as alert and list indices.
```

Reply:
I like that!